pkg/sippyserver/server.go

```diff
@@ -806,8 +808,19 @@ func (s *Server) jsonReleasesReportFromDB(w http.ResponseWriter, req *http.Reque
 
 	for _, release := range releases {
 		response.Releases = append(response.Releases, release.Release)
+		addDate := false
+		releaseDate := apitype.ReleaseDates{}
 		if release.GADate != nil {
-			response.GADates[release.Release] = *release.GADate
+			response.DeprecatedGADates[release.Release] = *release.GADate
+			releaseDate.GA = release.GADate
+			addDate = true
+		}
+		if release.DevelopmentStartDate != nil {
+			releaseDate.DevelopmentStart = release.DevelopmentStartDate
+			addDate = true
+		}
+		if addDate {
+			response.Dates[release.Release] = releaseDate
 		}
```

Comment:
total nit but it seems like unnecessary logic when you could just do
```suggestion
			response.Dates[release.Release] = releaseDate
		}
		if release.DevelopmentStartDate != nil {
			releaseDate.DevelopmentStart = release.DevelopmentStartDate
			response.Dates[release.Release] = releaseDate
		}
```